src/components/GrantRequest/ProjectDurationInput.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import Add from '../Icon/Add'
+import Remove from '../Icon/Remove'
+
+import Label from './Label'
+import './ProjectDurationInput.css'
+
+interface Props {
+  value: number
+  onChange: (value: number) => void
+  options: number[]
+}
+
+const ProjectDurationInput = ({ value, onChange, options }: Props) => {
+  const t = useFormatMessage()
+  const optionMinLimit = options[0]
+  const optionMaxLimit = options[options.length - 1]
+
+  const handleAddClick = useCallback(() => {
+    if (value === optionMaxLimit) {
+      return
+    }
+
+    onChange(value + 1)
+  }, [onChange, optionMaxLimit, value])
+
+  const handleRemoveClick = useCallback(() => {
+    if (value === optionMinLimit) {
+      return
+    }
+
+    onChange(value - 1)
+  }, [onChange, optionMinLimit, value])
+
+  return (
+    <div>
+      <Label>{t('page.submit_grant.funding_section.project_duration_title')}</Label>
+      <div>
+        <div className="ProjectDurationInput__InputContainer">
+          <div className="ProjectDurationInput__Input">
+            <button className="ProjectDurationInput__Button" onClick={handleRemoveClick}>
+              <Remove />
```

Comment:
WDYT about increase/decrease instead of add/remove?

Reply:
I think those are the common names in the Icons Realm. Other possibility could be using Minus / Plus

Reply:
I like better Increase/Decrease for this case. But I don't know if it's better to keep them more generic.